src/com/epam/javatrainings/homework_5/AsynchronousTimer.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.javatrainings.homework_5;
+import java.util.Scanner;
+import java.lang.Thread;
+import java.lang.Runnable;
+
+public class AsynchronousTimer {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        int minutes  = sc.nextInt();
+        int seconds = sc.nextInt();
```

Comment:
Should present some beginning message like(please enter preferable minute/second)
and also need to validate the seconds, value greater than 60 is not allowed.